video/out/wayland_common.c

```diff
@@ -251,7 +251,7 @@ static void pointer_handle_axis(void *data, struct wl_pointer *wl_pointer,
     struct vo_wayland_state *wl = data;
     if (wl_fixed_to_double(value) == 0)
         return;
-    double val = wl_fixed_to_double(value)/abs(wl_fixed_to_double(value));
+    double val = wl_fixed_to_double(value)/fabs(wl_fixed_to_double(value));
```

Comment:
Could be replaced with a ternary as noted in IRC. Also with that we could get rid of the above if line too.